.eslintrc.js

```diff
@@ -1,24 +0,0 @@
-module.exports = {
```

Comment:
Why was this file deleted?

Reply:
The recently merged PR must be showing conflicts now. Update your PR by resolving the conflicts.